### ISSUE TITLE
Update resources for all the reference tools

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#192]
[RawGit](https://rawgit.com/) is shutting down. And [OpenLayers](https://openlayers.org/en/latest/doc/quickstart.html) has shifted to using [jsdelivr](https://www.jsdelivr.com/) instead.

So we'll have to update the linked resources, and I propose using the latest version (which will need testing):

```diff
-https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/css/ol.css
+https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.0.1/css/ol.css
-https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/build/ol.js
+https://cdn.jsdelivr.net/gh/openlayers/openlayers.github.io@master/en/v6.0.1/build/ol.js
```

I also propose we use the latest version of resources for _all_ the reference tools, because it wouldn't be fair/accurate to assess older versions of a tool.

Obviously, it'd take a lot of work to _continuously_ update resources for all the reference tools and to also update the assessments based on a particular version, but I think we should considering updating resources at least once for now - as we seem to be using somewhat outdated resources in some cases (e.g. for CSS/JS we're using MapBox `v1.0.0` while latest is `v1.3.2`, or TomTom `v4.47.6` while latest seem to be `v5.36.1`).

/cc @AmeliaBR, @nchan0154, @NickFitz for thoughts.

**Comment by nchan0154** [comment]
When reviewing TomTom, I noticed v5 was listed as being under [public preview](https://developer.tomtom.com/public-preview) and that breaking changes were possible, which is why I opted to go with v4. I definitely support reviewing latest versions, but should this include latest 'unfinished' versions?

**Comment by Malvoz** [comment]
You did right doing so, and I think @AmeliaBR will want to avoid unstable releases as well.

**Comment by NickFitz** [comment]
I'll see about updating the existing examples (along with those in development) for OpenLayers, but leave the TomTom examples on v4 while v5 remains in preview.

**Comment by AmeliaBR** [comment]
Agreed. When we are finalizing the report for publication, we'll probably do a final review including identifying any updates in the tools that might affect conclusions. But I'm not going to tear everything out every time a minor version gets bumped up.

However, if the recommended CDN is shutting down, that's definitely something we need to fix, so might as well do an update at the same time.

**Comment by NickFitz** [comment]
FWIW, TomTom’s v5 is now generally available, with v4 reaching end of support in January 2021.
